frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -287,7 +287,7 @@ private void serveNewTableRowLinkedProjects(ResourceRequest request, ResourceRes
             for (String linkedId : linkedIds) {
                 Project project = client.getProjectById(linkedId, user);
                 ProjectLink linkedProject = new ProjectLink(linkedId, project.getName());
-                linkedProject.setRelation(ProjectRelationship.UNKNOWN);
+                linkedProject.setRelation(ProjectRelationship.REFERRED);
```

Comment:
But for projects the ticket says something: CONTAINED should be the default. Or do we have to change the ticket?